### PR TITLE
Apitest support http get

### PR DESCRIPTION
**Pull request by Max104t**
In order to properly test support for groups, this refactor to the `ApiTest` section will allow `v1` HTTP `GET` style calls..

Test get channels:

- Get authorization_token with `v2`
  - func: authorizeToken
  - param: {"service":"channels"}
- Get Channels with `v1`
  - url: https://channels.periscope.tv/v1/channels
  - HTML mthod `GET`
  - empty method, and params
  - copy the `authorization_token` to :
```JSON
headers: {
    "Authorization": "***", 
    "X-Periscope-User-Agent": "Periscope/2699 (iPhone; iOS 8.1.2; Scale/2.00)",
    "locale":"en"
}
```
  - run; you will get the broadcast channels

GET-API

### CODE REVIEW

**Comment by kewalsk** [comment]
Good, but other HTTP methods will be necessary also, not only GET.

**Comment by Max104t** [comment]
I've added all of them.. but I only tested POST/GET

METHOD | Description
---|---
GET|The GET method requests a representation of the specified resource. Requests using GET should only retrieve data.
HEAD|The HEAD method asks for a response identical to that of a GET request, but without the response body.
POST|The POST method is used to submit an entity to the specified resource, often causing a change in state or side effects on the server.
PUT|The PUT method replaces all current representations of the target resource with the request payload.
DELETE|The DELETE method deletes the specified resource.
CONNECT|The CONNECT method establishes a tunnel to the server identified by the target resource.
OPTIONS|The OPTIONS method is used to describe the communication options for the target resource.
TRACE | The TRACE method performs a message loop-back test along the path to the target resource.
PATCH | The PATCH method is used to apply partial modifications to a resource.

File: Periscope_Web_Client.user.js [hunk 1]
```diff
@@ -745,15 +749,18 @@ ApiTest: function () {
     submitButton.click(function () {
         try {
             $('#ApiTest form').submit();
+            var url_root = $('#url_root').val().trim();
+            if (url_root == '')
+                throw Error('url is empty');
+            var http_method = $('#http_method').val().trim();
             var method = $('#method').val().trim();
-            if (method == '')
```
Comment (gitnew2018) [inline Periscope_Web_Client.user.js]: Why remove this?

File: Periscope_Web_Client.user.js [hunk 2]
```diff
@@ -766,7 +773,19 @@ ApiTest: function () {
         $('<div id="ApiTest"/>').append(
             '<a href="https://github.com/gitnew2018/My-OpenPeriscope"><img style="position: absolute; top: 0; right: 0; border: 0;" src="' + IMG_PATH + '/images/forkme.png" alt="Fork me on GitHub"></a>' +
             'Some documentation can be found in <a href="http://static.pmmlabs.ru/OpenPeriscope" target="_blank">docs by @cjhbtn</a>' +
+            '<br/><dt>Url</dt><iframe id="urlautocomplete" name="urlautocomplete" style="display: none;"></iframe><form target="urlautocomplete"><input id="url_root" type="text" value="https://api.periscope.tv/api/v2/" autocomplete="on"/></input></form><br/>' +
```
Comment (gitnew2018) [inline Periscope_Web_Client.user.js]: Some <br>s could be removed, I used inspect tool to see which ones.

File: Periscope_Web_Client.user.js (hunk 1 above)
Comment (Max104t) [reply Periscope_Web_Client.user.js]: when we call `GET` the method the `URL` should be enough.. we could break it assuming that the method is going to be concatenated.. but this is a hidden information.. another way to do it is to disable the method when selecting `get` but I don't want to complicate things.. the call will `404` if the `URL` + `method` combo fails..

File: Periscope_Web_Client.user.js (hunk 2 above)
Comment (Max104t) [reply Periscope_Web_Client.user.js]: I removed 2 of them.. let me exactly which to remove and will take'm out..

**Pull request merged**